components/Modals/Videoinstructions/index.vue

```diff
@@ -0,0 +1,164 @@
+const isFormValid = ref<boolean>(false);
+const data = ref<IVideoinstructionPayloadCreate>({
+  title: props.videoinstruction?.title ?? "",
+  platform: props.videoinstruction?.platform ?? undefined,
+  video_url: props.videoinstruction?.video_url ?? "",
+});
+
+const platformItems = computed(() => {
+  if (!User.store.enums?.video_platforms) return [];
+
+  return Object.entries(User.store.enums.video_platforms).map(
+    ([value, title]) => ({
+      value: Number(value),
+      title,
+    }),
+  );
+});
+
+watch(dialog, (value) => {
+  emits("update:dialog", value);
+});
+
+watch(
+  () => props.dialog,
+  (value) => {
+    dialog.value = value;
+  },
+);
+
+const submitVideoinstruction = () => {
+  if (props.videoinstruction) {
+    return Videoinstruction.update({
+      id: props.videoinstruction.id,
+      ...data.value,
+    })
+      .then(() => {
+        toast.success("Видеоинструкция изменена");
+        emits("on-create");
+
+        dialog.value = false;
+      })
+      .catch((response) => {
+        useErrorNotification(response.response.data.errors);
+      });
+  }
+  return Videoinstruction.create(data.value)
+    .then(() => {
+      toast.success("Видеоинструкция создана");
+      emits("on-create");
+
+      dialog.value = false;
+    })
+    .catch((response) => {
+      useErrorNotification(response.response.data.errors);
+    });
+};
+</script>
+
+<template>
+  <component
+    :is="width > MOBILE_VERSION_WIDTH ? VDialog : VBottomSheet"
+    :max-width="width > MOBILE_VERSION_WIDTH ? 500 : MOBILE_VERSION_WIDTH"
+    v-model="dialog"
+    persistent
+  >
+    <template v-if="dialog">
+      <v-card
+        rounded="xl"
+        class="!bg-white dark:!bg-neutral-900 card-modal"
+        variant="outlined"
+        :loading="loading"
+      >
+        <v-card-title>
+          <div class="flex justify-between">
+            <div class="text-h5"> Создать видеоинструкцию </div>
+
+            <v-btn
+              icon="mdi-close"
+              :disabled="loading"
+              variant="text"
+              class="dark:text-white"
+              :ripple="false"
+              @click="dialog = false"
+            />
+          </div>
+        </v-card-title>
+
+        <v-card-text class="dark:text-white scrollable-content">
+          <v-form
+            v-model="isFormValid"
+            class="grid gap-5"
+            @submit.prevent="submitVideoinstruction"
+          >
+            <div class="flex flex-col gap-5">
+              <div>
+                <v-text-field
+                  hide-details="auto"
+                  rounded="lg"
+                  label="Название"
+                  :rules="Rule.getRequired()"
+                  variant="outlined"
+                  v-model="data.title"
+                />
+              </div>
+              <div>
+                <v-text-field
+                  v-model="data.video_url"
+                  :rules="[Rule.getRequired(), Rule.url]"
```

Comment:
Дважды написал и ты дважды ошибся.


```suggestion
                  :rules="[Rule.required, Rule.url]"
```

Не getRequired, а required